misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -1437,18 +1636,29 @@ public ByteArrayLiteralNode transform(ByteArrayLiteralNode byteArrayLiteralNode)
 
     @Override
     public ListConstructorExpressionNode transform(ListConstructorExpressionNode listConstructorExpressionNode) {
+        if (!isInLineRange(listConstructorExpressionNode, lineRange)) {
+            return listConstructorExpressionNode;
+        }
+        int newLines = 0;
+        if (listConstructorExpressionNode.children().get(1).kind().equals(SyntaxKind.MAPPING_CONSTRUCTOR)) {
```

Comment:
Whats the node we are getting here by `listConstructorExpressionNode.children().get(1)`?

Can we use the specific method in `ListConstructorExpressionNode` class?

Reply:
The expected node here is a `MappingConstructorExpressionNode`.

`ListConstructorExpressionNode` class returns a `SeparatedNodeList` as the set of expressions, so both ways would result in a similar code implementation AFAIU, but I will check on as to whether this could be improved.